src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/utils.ts

```diff
@@ -12,7 +12,7 @@ const getSafeVersion = (data) => {
 }
 
 interface TxData {
-  data?: string
+  data?: string | null
```

Comment:
All the other data definitions were changed to be `data: string | null`.

Is in this case data optional?

Reply:
I didn't do a deep dive, just made typescript happy

Reply:
I see. The initialization in the method below would fail because data is not defined on the base object.

I think this would worth a comment because in the rest of the app, similar props are mandatory. To be coherent and to make sense the use of typescript maybe it would need an initialization object that complies with the app most common types, making data mandatory.

Reply:
Yes I felt the same when fixing 350 errors in this PR :) But this is a little bit out of scope

Reply:
That component may need a deeper refactor, I'm checking it right now and there are at least 3 ```else``` setting the same data.

Too much else if, maybe a case should be used here But doesn't lock the PR